docs/02-app/01-building-your-application/11-upgrading/02-version-15.mdx

```diff
@@ -25,7 +25,10 @@ pnpm up next@rc react@rc react-dom@rc eslint-config-next@rc
 bun add next@rc react@rc react-dom@rc eslint-config-next@rc
 ```
 
-> **Good to know:** If you are using TypeScript, ensure you also upgrade `@types/react` and `@types/react-dom` to their latest versions.
+> **Good to know:**
+>
+> - If you see a peer dependencies warning, you may need to update `react` and `react-dom` to the suggested versions, or you use the `--force` or `--legacy-peer-deps` flag to ignore the warning. This won't be necessary once both Next.js 15 and React 19 are stable.
+> - If you are using TypeScript, ensure you also upgrade `@types/react` and `@types/react-dom` to their latest versions.
```

Comment:
The release blog post from react recommended types-react and types-react-dom, is that still the case?

- https://react.dev/blog/2024/04/25/react-19-upgrade-guide#installing
- https://www.npmjs.com/package/types-react/v/19.0.0-rc.1
- https://www.npmjs.com/package/types-react-dom/v/19.0.0-rc.1

Reply:
Linked to the React RC upgrade guide.